.github/pull_request_template.md

```diff
@@ -1,6 +1,8 @@
 #### Description of changes
 
-(give an overview. Add technical description describing your changes)
+<!-- give an overview. Add technical description describing your changes -->
+
+<!-- Please use a suitable tag (feat, fix, chore, refactor) on the PR message or PR title to help in release validation and release notes -->
```

Comment:
I think we mentioned to make this one of the bullet points on the check list at the bottom of the PR template

Reply:
Updated.